aquadoggo/src/graphql/client/mutation.rs

```diff
@@ -338,128 +299,128 @@ mod tests {
     #[rstest]
     #[case::no_entry(
         "",
-        "",
-        "Failed to parse \"EntrySignedScalar\": Bytes to decode had length of 0"
+        &OPERATION_ENCODED,
+        "Bytes to decode had length of 0"
     )]
     #[case::invalid_entry_bytes(
         "AB01",
-        "",
-        "Failed to parse \"EntrySignedScalar\": Could not decode author public key from bytes"
+        &OPERATION_ENCODED,
+        "Could not decode author public key from bytes"
     )]
     #[case::invalid_entry_hex_encoding(
         "-/74='4,.=4-=235m-0   34.6-3",
         &OPERATION_ENCODED,
-        "Failed to parse \"EntrySignedScalar\": invalid hex encoding in entry"
+        "Failed to parse \"EntrySignedScalar\": Invalid character '-' at position 0"
     )]
     #[case::no_operation(
-        &ENTRY_ENCODED,
-        "",
-        "operation needs to match payload hash of encoded entry"
+        &EncodedEntry::from_bytes(&ENTRY_ENCODED).to_string(),
+        "".as_bytes(),
+        "cbor decoder failed failed to fill whole buffer"
```

Comment:
Is this the error we expect for this case? It seems to come from a cbor IO error: https://github.com/p2panda/p2panda/blob/3ea76a53bac01a073c226cd4b64085a58bf14ae2/p2panda-rs/src/operation/decode.rs#L35

Reply:
Oh yeah, the error makes sense but the message is not so nice. We shouldn't change the IO error message but we can check if the array is empty before it arrives there.